src/frontends/pytorch/src/op/adaptive_max_pool2d.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_adaptive_max_pool2d(NodeContext& context) {
+    auto x = context.get_input(0);
+    auto y = context.get_input(1);
+    auto adaptive_max_pool = context.mark_node(std::make_shared<opset10::AdaptiveMaxPool>(x, y, ov::element::i32));
+    return {adaptive_max_pool->output(0), adaptive_max_pool->output(1)};
```

Comment:
please check this implementation because it is much different from `translate_adaptive_avg_pool3d`: different number of outputs, transformation.

Reply:
Same